src/hooks/useCedarContactLookup.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useApolloClient } from '@apollo/client';
+
+import GetCedarContactsQuery from 'queries/GetCedarContactsQuery';
+import { CedarContactProps } from 'types/systemIntake';
+
+function useCedarContactLookup(
+  query: string
+): { [id: string]: CedarContactProps };
+
+// eslint-disable-next-line no-redeclare
```

Comment:
There's a `typescript-eslint` rule that you can enable that takes into account TS function overloading; see https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-redeclare.md. Add that to `.eslintrc` inside the `"rules"` section, and you shouldn't need these eslint-disables.